### ISSUE TITLE
Portrait Mode?

### ISSUE THREAD

**Issue by ad-borges** [blakadder/nspanel#33]
I have the US version
How can i use portrait mode? (vertical)
Any command?

**Comment by joBr99** [comment]
if you have the us version, you should should be able to change orientation in the settings on the panel itself

**Comment by ad-borges** [comment]
Already flashed to Blackadder's tasmota code.... 
Not stock firmware anymore

All that appears in screen right now, is commanded by driver.... at config page, theres Brightness setable, theres sleep setable. Orientation option, when i click in Portrait, nothing happens, because its not mapped

**Comment by joBr99** [comment]
press restart in tasmota after you change the orientation on nspanel

**Comment by ad-borges** [comment]
The choice input radio is not selectable. I click in portrait, dont get blue, radio button not change from landscape to portrait.

**Comment by ad-borges** [comment]
Even not changing in screen, i restarted and worked..... thanks mate!!!!

**Comment by joBr99** [comment]
yes, the firmware on the display of nspanel is setting some magic numbers somewhere in the firmware, causing it to load an alternative firmware for the other orientation on next boot (and it should also send a msg to esp32 to request a power reset)

but this message isn't doing anthing, because that's not implemented in the berry driver

**Comment by ad-borges** [comment]
@joBr99 you can point me to some resource regarding node-red logic to manage nspanel state syncs and controls?
i use mostly zigbee with mqtt and homebridge, so i never adopted Home Assistant

**Comment by joBr99** [comment]
sorry, can't help you with that, not using the stock fw

**Comment by ad-borges** [comment]
thanks

**Comment by ad-borges** [comment]
@joBr99 Last one :)
Whats the difference between Analog and ESP-32 Temperature? The last one is about the chip itself?

**Comment by joBr99** [comment]
yep, analog1 is the ntc thermistor for the room temp, esp32 temp is the chip itself
